### PR TITLE
Modify files in Docker directory - change rolling to humble

### PR DESCRIPTION
**Pull request by Xinrt**
In Docker/Dockerfile, the version used in branch humble is rolling. And the clone request is based on the master branch.

Then, I changed the code:
1. change `rolling` to `humble`
2. change `git clone` to `git clone -b humble`

### CODE REVIEW

**Comment by bmagyar** [comment]
Solves #189

**Review by bmagyar — APPROVED** [review]

File: .DS_Store
Comment (bmagyar) [inline .DS_Store:1]: Please don't add local garbage to the PR

Comment (Xinrt) [reply .DS_Store:1]: Ok, I'll pay attention to that next time.

Comment (bmagyar) [reply .DS_Store:1]: I won't take that as an answer ;) REmoved it from the branch

**Review by bmagyar — APPROVED** [review]

**Pull request merged**